lib/containers/k8s.pm

```diff
@@ -103,6 +103,8 @@ sub install_k3s {
         delete $k3s_args{$key};
     }
 
+    zypper_call('in k3s-selinux') if is_tumbleweed;
```

Comment:
Is `is_tumbleweed` the right condition for this?

Reply:
> Is `is_tumbleweed` the right condition for this?

What do you suggest?

Reply:
If it's a workaround for https://github.com/k3s-io/k3s/issues/10876 then `is_tumbleweed && !is_microos` would probably work?

Reply:
Makes sense.